src/Umbraco.Web/Security/MembershipHelper.cs

```diff
@@ -289,11 +289,46 @@ public virtual IPublishedContent GetByProviderKey(object key)
             return MemberCache.GetByProviderKey(key);
         }
 
+        public virtual IEnumerable<IPublishedContent> GetByProviderKey(IEnumerable<object> keys)
+        {
+            return keys?.Select(GetByProviderKey).WhereNotNull() ?? new IPublishedContent[0];
+        }
+
+        public virtual IEnumerable<IPublishedContent> GetByProviderKey(params object[] keys)
+        {
+            return keys?.Select(GetByProviderKey).WhereNotNull() ?? new IPublishedContent[0];
+        }
+
         public virtual IPublishedContent GetById(int memberId)
         {
             return MemberCache.GetById(memberId);
         }
 
+        public virtual IEnumerable<IPublishedContent> GetById(IEnumerable<int> memberIds)
```

Comment:
For methods that return multiple items, should we make these method names plural like `GetByIds` ?

Reply:
I really really wanted to do that when I did the PR. The reason I didn't is that the similar overloads on `UmbracoHelper` do not have plural notation - e.g. `Content(int id)` and `Content(params object[] ids)`. But... when digging further it's evident that plural notation is used elsewhere , like in `IContentService.GetByIds(IEnumerable<int> ids)`, so it seems feasible to do it here as well. I'll update the PR.

Reply:
Fixed in 64bae43